### ISSUE TITLE
Add new facet to AAPB: "Materials Available for Reuse"

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH-MLA/AAPB2#1113]
Some donors have begun to dedicate their materials to a creative commons license. We would like to create a new facet on the AAPB to allow users to filter their results to materials available for reuse. 

Some CC licenses allow for broad reuse, while others allow for limited reuse. Europeana has created facets to this effect: http://www.europeana.eu/portal/en/search?q=dogs

We will need to add any Creative Commons license information to the AAPB metadata records for the applicable assets. @sroosa what do you propose for the metadata field? 

**waiting on:**
  - [ ] AMS is updated with new metadata re: licenses
  - [ ] developers are instructed on which fields to look for

**Done when:**
   - [ ] metadata field to track license info is added to PBCore models
   - [ ] metadata is successfully imported from AMS to solr
   - [ ] affected records are re-indexed

**Comment by sroosa** [comment]
@caseyedavis12 : I think we should use the pbcoreRightsSummary - rightsLinks subelement.

**Comment by afred** [comment]
@sroosa @caseyedavis12 do we want to do add some one-off metadata specific to Creative Commons, or are we talking about a more generic field for having different licenses?

**Comment by caseyedavis12** [comment]
@afred: I was thinking this would be Creative Commons licenses (there are 6, but we currently have collections pursuant to 3 of them) plus public domain.

**Comment by sroosa** [comment]
Duplicate of #1463
